src/components/node-list/node-list-items.test.js

```diff
@@ -50,7 +50,7 @@ describe('node-list-selectors', () => {
 
     it('returns expected number of tags', () => {
       expect(filteredTags.length).not.toBe(tags.length);
-      expect(filteredTags).toHaveLength(2);
+      expect(filteredTags).toHaveLength(1);
```

Comment:
Could we use a search with 2 or more results ideally?

Reply:
updated the search value to generate more results for this test!